### PR TITLE
[MM-14018] Ignore leading/trailing colon in emoji picker search

### PR DESCRIPTION
**Pull request by eshyong**
#### Summary
Creates a helper function which is called in the EmojiPicker component and strips leading/trailing colons from the input.

#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/MM-14018
https://github.com/mattermost/mattermost-server/issues/10276

#### Related Pull Requests
None

### CODE REVIEW

File: components/emoji_picker/emoji_picker.jsx [hunk 1]
```diff
@@ -112,6 +112,10 @@ function getEmojiFilename(emoji) {
     return emoji.filename || emoji.id;
 }
 
+export function filterEmojiSearchInput(input) {
+    return input.toLowerCase().replace(/^:/g, '').replace(/:$/g, '');
```
Comment (saturninoabril) [inline components/emoji_picker/emoji_picker.jsx]: I think the regex can be combined, something like,
```suggestion
    return input.toLowerCase().replace(/^:|:$/g, '');
```

**Comment by eshyong** [comment]
@saturninoabril created another PR [here](https://github.com/mattermost/mattermost-mobile/pull/2974) for the mobile repo

**Comment by eshyong** [comment]
@devinbinnie still need approval from you

**Comment by eshyong** [comment]
Thanks Devin. @hanzei how should I get this merged?

**Comment by hanzei** [comment]
@eshyong I'm just going to merge this. Thanks for the ping :+1:

**Pull request merged**